### ISSUE TITLE
Broken Backward compatibility with ControllerArgs

### ISSUE THREAD

**Issue by DavidMansolino** [cyberbotics/webots#1843]
**Describe the Bug**
In some cases (e.g. procedural PROTOs) the change of the `controllerArgs` fields from SFString to MFString broke the backward compatibility.

For example, with such PROTO:
```
#VRML_SIM R2020a utf8
# tags: static

PROTO MyCar [
  field SFVec3f      translation              0 0.4445 0
  field SFRotation   rotation                 0 1 0 0
  field SFColor      color                    0.92 0.94 0.94
  field SFString     controller               "void"
  field SFString     controllerArgs           ""
]
{
  Car {
    translation IS translation
    rotation IS rotation
    controller IS controller
    controllerArgs IS controllerArgs
    recognitionColors [
      %{= fields.color.value.r }% %{= fields.color.value.g }% %{= fields.color.value.b }%
    ]
  }
}
```

2 controllerArgs fields are generated in the scene-tree (without any warnings):
![aaa](https://user-images.githubusercontent.com/2461619/85988364-33774c00-b9ef-11ea-91d0-7c346278f2fd.png)

None of them is usable (not passed to the controller).

**Comment by omichel** [comment]
I believe this is more a general PROTO checking issue: this [method](https://github.com/cyberbotics/webots/blob/master/src/webots/vrml/WbProtoModel.cpp#L457) is not called on the MyCar device (it should check the match of the IS data types).

**Comment by omichel** [comment]
Fixed in #1847
